tests/test_get_component_info.py

```diff
@@ -45,6 +51,14 @@ def check_result(self, result_path: pathlib.Path, name: str) -> None:
                     avro_schema = json.loads(fp.read())
                     result["topics"][avro_schema["name"]] = avro_schema
 
+        assert hash_table_created
+
+        xml_type_expected = {"Commands", "Events", "Generics"}
```

Comment:
Is telemetry handled outside this?

Reply:
The problem is that not all components have telemetry.. :(  Script, which is used for testing does not.

Reply:
OK. Thanks for the explanation.